Packs/Active_Directory_Query/Scripts/IAMInitADUser/IAMInitADUser.py

```diff
@@ -169,15 +226,115 @@ def create_zip_with_password(generated_password, zip_password):
             zip_content = zip_file.read()
 
     except Exception as e:
-        raise Exception(f'An error occurred while trying to create a zip file. Error is:\n{str(e)}')
+        raise DemistoException(f'Could not generate zip file. Error:\n{str(e)}')
 
     finally:
         for file_name in (text_file_name, zip_file_name):
             if os.path.exists(file_name):
                 os.remove(file_name)
 
-    return zip_content
+    return_results(fileResult(zip_file_name, zip_content))
+
+    return PollResult(
+        response=None,
+        continue_to_poll=True,
+        partial_result=CommandResults(readable_output=f"Encrypted zip file generated. File name: '{zip_file_name}'."),
+        args_for_next_run={**args, 'zip_file_name': zip_file_name},
+    )
```

Comment:
Returning a list of `[fileResult(), PollResult()]` from the function (which would look more in-convention) unfortunately doesn't work, as the `polling_function` decorator requires the function to return a `PollResult` (returning a list causes it to fail).

Reply:
Please open a dev bug for this.